### PR TITLE
Update dependency versions from Jet 4.5 made after the merge

### PR DESCRIPTION
**Pull request by frant-hartm**
This also adds dependency convergence task to check the dependecies
resolve to the same version in the extensions. This avoids having
different extensions built with the same dependency, but a different
version. The extensions are on the same classpath, which could cause
random and hard to debug CP issues.

This also reorders the version properties alphabetically.

### CODE REVIEW

**Review by kwart — APPROVED** [review]
LGTM

**Review by vbekiaris — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -1231,6 +1215,13 @@
                 <scope>import</scope>
                 <type>pom</type>
             </dependency>
+            <dependency>
```
Comment (mmedenjak) [inline pom.xml:1218]: What's this change about? cc @kwart @vbekiaris

Comment (kwart) [reply pom.xml:1218]: The Bill Of Materials in the dependencyManagement allows defining one umbrella dependency for all netty components.

Comment (mmedenjak) [reply pom.xml:1218]: Thanks, just double-checking we're fine with it. Merging with unrelated test failure.

**Pull request merged**